src/detail-page/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  Accordion as MuiAccordion,
+  AccordionDetails as MuiAccordionDetails,
+  AccordionSummary as MuiAccordionSummary,
+  List,
+  Typography,
+} from '@material-ui/core';
+import { ExpandMore } from '@material-ui/icons';
+import { AccordionListItem } from './AccordionListItem';
+import { Resource } from '../../../data/detail-page/resources/ResourcesInterfaces';
+
+interface AccordionProps {
+  header: string;
+  list: Resource[];
+  panelNumber: number;
+}
+
+const StyledAccordion = styled(props => <MuiAccordion {...props} />)`
```

Comment:
I admit that it is a kind of magic for me a little. :D
In `Accordion` & `AccordionSummary` I added passing props through function when I experimented with other ways of styling (passing `expanded` props instead of classes). Now with class styling I removed it and it looks all right.
But in `StyledListItem` component (which is styled `ListItem` component) when I remove function I get typescript error.
![list-item](https://user-images.githubusercontent.com/29304012/115905869-0658c600-a467-11eb-9ec4-a0502b2a166d.png)